src/ReflectionClassConstant.php

```diff
@@ -164,33 +165,41 @@ public function getValue()
     }
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
      */
-    public function isPrivate()
+    public function isPrivate(): bool
     {
         return $this->classConstantNode->isPrivate();
     }
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
      */
-    public function isProtected()
+    public function isProtected(): bool
     {
         return $this->classConstantNode->isProtected();
     }
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
      */
-    public function isPublic()
+    public function isPublic(): bool
     {
         return $this->classConstantNode->isPublic();
     }
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
+     */
+    public function getAttributes(?string $name = null, int $flags = 0): array
+    {
+        return parent::getAttributes($name, $flags);
```

Comment:
Unfortunately, this violates main principle of library - to avoid loading of class during analysis. 

This part requires some code, which **uses only AST information from node** to emulate original behavior. Should be relatively easy to implement starting from `nikic/php-parser >= 4.10.0`, see https://github.com/nikic/PHP-Parser/releases/tag/v4.10.0